django/core/files/move.py

```diff
@@ -82,12 +81,15 @@ def file_move_safe(
 
     try:
         copystat(old_file_name, new_file_name)
-    except PermissionError as e:
+    except PermissionError:
         # Certain filesystems (e.g. CIFS) fail to copy the file's metadata if
         # the type of the destination filesystem isn't the same as the source
-        # filesystem; ignore that.
-        if e.errno != errno.EPERM:
-            raise
+        # filesystem. This also happens with some SELinux-enabled systems.
+        # Ignore that, but try to set basic permissions.
+        try:
+            copymode(old_file_name, new_file_name)
+        except PermissionError:
+            pass
```

Comment:
I think it's safe to ignore any error in copymode since a file already copied.  
The previous behavior worked around exception in `utime` call and copymode doesn't contains such call.

Reply:
Ok :+1: